examples/kitchen-sink.yaml

```diff
@@ -199,6 +203,10 @@ meter_provider:
             #
             # Environment variable: OTEL_EXPORTER_OTLP_INSECURE, OTEL_EXPORTER_OTLP_METRICS_INSECURE
             insecure: false
+            # Configure retry policy
+            retry:
+              # Configure retry enabled
```

Comment:
Not sure this is the PR to solve it, but we might need to come up with a strategy for making it clear what the default values are. Our guidance in the [spec is](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/file-configuration.md#create):

> If a field is null or unset and a default value is defined, Create MUST ensure the SDK component is configured with the default value. 

In every case so far, its always been clear what the default is because we have a policy [that says we only include properties](https://github.com/open-telemetry/opentelemetry-configuration?tab=readme-ov-file#what-properties-are-part-of-schema) in the schema which are part of the spec. This means SDKs can always check the spec to see what the default is. 

In the case of retry enabled, the [spec says](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#retry): 

> Transient errors MUST be handled with a retry strategy. 

This suggests that `*.retry.enabled=true` by default, but I'm not sure that's the case in most SDK implementations.

Reply:
Opened https://github.com/open-telemetry/opentelemetry-specification/pull/4148 to explicitly define a corresponding option for this in the spec along with a default.